front/gatsby/src/components/Layout/Header/Header.tsx

```diff
@@ -159,17 +163,17 @@ const StakingHeader = () => {
       if (!stakeHead || stakeHead.blockNumber > number) {
         setStakeHead({
           blockNumber: number,
-          totalStake,
+          totalStake: api.createType('Balance', totalStake),
```

Comment:
@amaurymartiny true, but https://github.com/paritytech/Nomidot/pull/169#discussion_r378940317 was still fresh in my mind 😅